phenotools-cli/src/main/java/org/phenopackets/phenotools/examples/Retinoblastoma.java

```diff
@@ -0,0 +1,321 @@
+import org.phenopackets.phenotools.builder.PhenopacketBuilder;
+import org.phenopackets.phenotools.builder.builders.*;
+import org.phenopackets.schema.v2.Phenopacket;
+import org.phenopackets.schema.v2.core.*;
+
+import java.util.List;
+
+import static org.phenopackets.phenotools.builder.builders.OntologyClassBuilder.ontologyClass;
+import static org.phenopackets.phenotools.builder.builders.Util.*;
+
+public class Retinoblastoma implements PhenopacketExample {
+    private static final String PHENOPACKET_ID = "arbitrary.id";
+    private static final String PROBAND_ID = "proband A";
+    private static final String BIOSAMPLE_ID = "biosample.1";
+    private static final OntologyClass BIOPSY = ontologyClass("NCIT:C15189", "Biopsy");
+    private static final OntologyClass retinoblastoma = ontologyClass("NCIT:C7541", "Retinoblastoma");
+
+    private final OntologyClass leftEye = ontologyClass("UBERON:0004548", "left eye");
+
+    private final Phenopacket phenopacket;
+    public Retinoblastoma() {
+        var metadata = MetaDataBuilder.create("2021-05-14T10:35:00Z", "anonymous biocurator")
+                .resource(Resources.ncitVersion("21.05d"))
+                .resource(Resources.efoVersion("3.34.0"))
+                .resource(Resources.uberonVersion("2021-07-27"))
+                .resource(Resources.ncbiTaxonVersion("2021-06-10"))
+                .build();
+        Individual proband = IndividualBuilder.create(PROBAND_ID).
+                ageAtLastEncounter("P6M").
+                female().
+                XX().
+                build();
+        // VCF file with results of germline whole-genome sequencing
+        File wgsFile = FileBuilder.file("file://data/germlineWgs.vcf.gz");
+
+        phenopacket = PhenopacketBuilder.create(PHENOPACKET_ID, metadata)
+                .individual(proband)
+                .allMeasurements(getMeasurements())
+                .allPhenotypicFeatures(getPhenotypicFeatures())
+                .disease(getDisease())
+                .medicalAction(melphalan())
+                .medicalAction(chemoRegimen())
+                .medicalAction(enucleation())
+                .biosample(enucleatedEye())
+                .interpretation(interpretation())
+                .file(wgsFile)
+                .build();
+    }
+
+
+    Interpretation interpretation() {
+        InterpretationBuilder ibuilder = InterpretationBuilder.solved("interpretation.id");
+        GenomicInterpretation somatic = somaticRb1Missense();
+         DiagnosisBuilder dbuilder = DiagnosisBuilder.create(retinoblastoma);
+         dbuilder.genomicInterpretation(somatic);
+         dbuilder.genomicInterpretation(germlineRb1Deletion());
+         ibuilder.diagnosis(dbuilder.build());
+         return ibuilder.build();
+    }
+
+
+    /**
+     * Variation ID:126824 (ClinGen pathogenic)
+     * @return Genomic interpretation related to a somatic missense mutation in the RB1 gene.
+     */
+    GenomicInterpretation somaticRb1Missense() {
```

Comment:
How do I know this is a somatic variant? Which statement points to that (I'm just curious)?